lib/jekyll/drops/drop.rb

```diff
@@ -137,6 +137,7 @@ def inspect
         require 'json'
         JSON.pretty_generate to_h
       end
+      alias_method :to_json, :inspect
```

Comment:
I mimicked the method aliasing done for the `to_h` method. But this is not part of the PR now anyway ;)